### ISSUE TITLE
How to use intcal20 for calibration?

### ISSUE THREAD

**Issue by benmarwick** [ropensci/c14bazAAR#118]
I was expecting to use `...` to pass the calibration curve to ` Bchron::BchronCalibrate()` like so:

```
library(c14bazAAR)
adrac <- get_c14data("adrac")
Batalimo <- adrac %>%
  dplyr::filter(site == "Batalimo")
calibrate(Batalimo,
          choices = "calprobdistr",
          calCurves = rep("intcal20", length(Batalimo$c14age)))
```

But I get this error:

```
Calibrating dates... 
  |++                                                |   5%Error in Bchron::BchronCalibrate(ages = x$c14age, ageSds = x$c14std, calCurves = rep("intcal13",  : 
  formal argument "calCurves" matched by multiple actual arguments
```

Am I doing something wrong? Thank you.

**Comment by nevrome** [comment]
That was not possible so far, because the calibration curve was hard-wired. Should be fixed with 569c0453bdea0ea984cbb06c61d06b298fccb529. Please test.

**Comment by benmarwick** [comment]
Perfect! Thank you so much for your quick action and reply!
